pom.xml

```diff
@@ -78,6 +78,7 @@
         <module>powerauth-fido2</module>
         <module>powerauth-java-server</module>
         <module>powerauth-admin</module>
+        <module>powerauth-fido2-demo</module>
```

Comment:
Not crucial, but wondering whether we should not use alphabetical order or at least group fido functionality together.

Reply:
good idea